draft-ietf-rats-architecture.md

```diff
@@ -405,21 +482,21 @@ generates the Evidence of the whole Composite Attester.
 
 The Lead Attester's Attesting Environment may or may not include its own
 Verifier.
-One situation is that the Attesting Environment has no local Verifier.
-In this situation, the Lead Attesting Environment simply combines the various
+The first variant is that the Attesting Environment has no local Verifier.
+In this variant, the Lead Attesting Environment simply combines the various
 Evidences into the final Evidence that is sent off to the remote Verifier,
-which evaluates the trustworthiness of the Composite Device,
-including the Lead Attester's and other Attesters', trustworthiness.
+which appraises the trustworthiness of the Composite Device,
+including the Lead Attester's and other Attesters' trustworthiness.
 
-The other situation is that the Lead Attesting Environment has an local Verifier.
+The second variant is that the Lead Attesting Environment has a local Verifier.
 After collecting the Evidence of other Attesters, this Attesting Environment
-verifies them using Endorsements and Appraisal Policies (obtained the
-same way as any other Verifier), for evaluating these Attesters' trustworthiness.
-Then the Lead Attesting Environment combines the Attestation Results into
+appraises them using Endorsements and Appraisal Policies (obtained the
+same way as any other Verifier).
+The Lead Attesting Environment combines the Attestation Results into
 the final Evidence of the whole Composite Attester which is sent off to the remote
-Verifier, which might treat the claims obtained from the local Attestation Results
+Verifier, which may treat the Claims obtained from the local Attestation Results
```

Comment:
Disagree with changing "might" to "may" which sounds too close to MAY which has a very different meaning, as opposed to "might" which is unambiguous.

Reply:
According to the new requirements notation a 'may' may mean 'MAY'. Reverted back to might to avoid any ambiguity here.